app/models/podcast_import.rb

```diff
@@ -148,6 +149,26 @@ def create_podcast
     podcast
   end
 
+  def enclosure_prefix(podcast_item)
+    prefix = ''
+    link = [podcast_item.feedburner_orig_enclosure_link,
```

Comment:
your way totally works, but as we old perl hackers like to say [TMTOWTDI](https://en.wikipedia.org/wiki/There's_more_than_one_way_to_do_it)